### PR TITLE
Disclosure Panel and Banner

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

A new page has been created to add a custom message to the specific places of the Chef Automate interface:
- A Disclosure Panel
- Banner

### :chains: Related Resources

The task is related to:
- https://github.com/chef/automate/pull/5106
- https://github.com/chef/automate/pull/5106
- https://github.com/chef/automate/issues/4934

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/configuration.md [hunk 1]
```diff
@@ -452,6 +452,36 @@ Configuration to sign out users from Chef Automate when they close the browser.
...
+    message = "Lorem ipsum dolor sit amet" # Add the Message for the banner
+    background_color = "3864f2" # Set the background color using the Hex Color Code (Do not add # to the code)
+    text_color = "FFF" # Set the color of the text using the Hex Color Code (Do not add # to the code)
+
+    # Find valid HEX codes at https://htmlcolorcodes.com/
+```
+
+#### Disclosure Panel
+
+Configuration to display a disclosure on the sign in page. Requires a `.txt` or `.html` message stored in a location accessible to Chef Automate. Default: false
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/configuration.md:473]:
```suggestion
Configuration to display a disclosure on the sign-in page. Requires a `.txt` or `.html` message stored in a location accessible to Chef Automate. Default: `false`.
```

File: components/docs-chef-io/content/automate/configuration.md [hunk 2]
```diff
@@ -452,6 +452,36 @@ Configuration to sign out users from Chef Automate when they close the browser.
   persistent = false
 ```
 
+#### Disclosure Banner
+
+Configuration to display a custom banner on every Chef Automate page including the sign in page. Default: false
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/configuration.md:457]:
```suggestion
Configuration to display a custom banner on every Chef Automate page, including the sign-in page. Default: `false`.
```

File: components/docs-chef-io/content/automate/disclosure_panel_and_banner.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+[menu]
+  [menu.automate]
+    title = "Disclosure Panel and Banner"
+    parent = "automate/configuring_automate"
+    identifier = "automate/configuring_automate/disclosure_panel_and_banner.md Disclosure Panel and Banner"
+    weight = 60
++++
+
+Customize your Chef Automate installation with a **Disclosure Panel** to add critical information about your unique environment on the sign-in page and a **Banner** at the top of every page to highlight important information.
+
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/disclosure_panel_and_banner.md:17]:
```suggestion
```

File: components/docs-chef-io/content/automate/disclosure_panel_and_banner.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## Disclosure Panel
+
+The **Disclosure Panel** is a space where you can add custom content to the Chef Automate sign-in page. Use this panel to display information, such as a message, warning, or instructions, that is during the sign-in process.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/disclosure_panel_and_banner.md:21]:
```suggestion
The **Disclosure Panel** is a space where you can add custom content to the Chef Automate sign-in page. Use this panel to display information, such as a message, warning, or instructions, that appears during the sign-in process.
```

File: components/docs-chef-io/content/automate/disclosure_panel_and_banner.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    show = true # Set 'show' to 'true' to enable the disclosure panel on the sign-in page. Set to 'false' to disable the disclosure panel. Default: false.
+
+    message_file_path = "/src/dev/disclosure-panel-message.txt" # The location of the file containing the disclosure panel message.
+
+    # Validate your HTML at https://validator.w3.org/
+```
+
+## Banner
+
+The  **Banner** displays custom content at the top of every page Chef Automate, including the sign-in page. Use it to add a message, warning, or instructions to Chef Automate users.The Banner can display only one message, warning or an instruction throughout the interface.
```
Comment (IanMadd) [inline components/docs-chef-io/content/automate/disclosure_panel_and_banner.md:45]:
```suggestion
The  **Banner** displays custom content at the top of every page in Chef Automate, including the sign-in page. Use it to add a message, warning, or instructions to Chef Automate users. The Banner can display only one message, warning or instruction throughout the interface.
```

File: components/docs-chef-io/content/automate/disclosure_panel_and_banner.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    show = true # Set 'show' to 'true' to enable the disclosure panel on the sign-in page. Set to 'false' to disable the disclosure panel. Default: false.
+
+    message_file_path = "/src/dev/disclosure-panel-message.txt" # The location of the file containing the disclosure panel message.
+
+    # Validate your HTML at https://validator.w3.org/
+```
+
+## Banner
+
+The  **Banner** displays custom content at the top of every page in Chef Automate, including the sign-in page. Use it to add a message, warning, or instructions to Chef Automate users. The Banner can display only one message, warning or instruction throughout the interface.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/disclosure_panel_and_banner.md:44]:
```suggestion
The **Banner** displays custom content at the top of every page in Chef Automate, including the sign-in page. Use it to add a message, warning, or instructions to Chef Automate users. The Banner can display only one message, warning or instruction throughout the interface.
```

**Review by kagarmoe — APPROVED** [review]
tenor-123834029

**Review by kalroy — APPROVED** [review]
Looks good. Thank you!!

**Pull request merged**